packages/namekit-react/src/components/DisplayedPrice.tsx

```diff
@@ -28,26 +28,26 @@ interface DisplayedPriceProps {
   // Wether to display the alternative price as a text instead of a tooltip
   altPriceDisplayFormat: AltPriceDisplayFormat;
   // Wether to display a currency symbology as its acronym or symbol
-  currencySymbology?: PriceSymbology;
+  currencySymbology?: CurrencySymbology;
   // The place to display the currency symbology
-  symbolPosition?: PriceSymbolPosition;
+  symbolPosition?: CurrencySymbolPosition;
   // The size of the alternative price display
   altPriceDisplaySize?: PriceDisplaySize;
   // The size of the price display
   priceTextDisplaySize?: PriceDisplaySize;
   // The place to display the alternative price when displayed as text
   altPriceDisplayPosition?: PriceDisplayPosition;
   // Wether or not to display the name of the currency in a tooltip when its symbol is hovered
-  showCurrencyTooltipDescription?: boolean;
+  describeCurrencyInTooltip?: boolean;
 }
 
 export const DisplayedPrice = ({
   price,
   altPrice,
   onTextClick,
```

Comment:
Semantically, `DisplayedPrice` is a text. The `onTextClick` possibility is an additional feature for integrators that want to give it functionality. Your idea is good but does not match all use cases, e.g. plain text `DisplayedName`

Reply:
@FrancoAguzzi But if I want to make a DisplayedPrice clickable, why wouldn't I just nest it inside of a <a or <Link tag?

@notrab Could you please share your advice here too?

Reply:
@FrancoAguzzi Hey great questions :+1:

1. Possibly. In my thinking, if some situation wanted the DisplayedPrice to be clickable, that should be achieved through an <a or a <Link that wraps the DisplayedPrice. This is outside my expertise though so please push back on this if I'm missing something. Something just doesn't feel right to me when we keep defining all these custom onClick handlers. My intuition is that this is not the right path.
2. Same as above.
3. They can be the same.

I think we'll need to put some boundaries on the customizations we support on these UI components. Without some boundaries the scope and complexity for us grows too much. Additionally, that complexity can make the UI components less easy to use or for people to feel comfortable using the library because it seems too complex.

There's a natural tension between these competing goals.

A good strategy can be to write our code in more composable ways. For example, if we have:

1. A UI component for displaying a primary price
2. A UI component for displaying an alt price
3. A UI component that wraps both of the above to display both a primary and alt price.

Then, in the usual case, people can just use UI component 3. In a more advanced case where someone wants to write different click handlers for primary price vs alt price, they can still make use of 1 and 2.

I appreciate that a refactoring as described here adds scope. If you think this direction makes sense then please create a new and distinct shortcut story for this as we can come back to it later in a separate PR.

Appreciate your advice :+1:

Reply:
I really appreciate your advices shared above, @lightwalker-eth 

Understanding more the time precision, I think it is great idea to wrap these in a Shortcut story and replicate the idea, in code format, soon.

Goal for now is, then, to make DisplayedPrice perfect for merge,
Next goal is to properly create the possibility of destructuring DisplayedPrice into single UI components and maturing these single implementation to the space where these can be used in pairs as well.

This is the [Story](https://app.shortcut.com/ps-web3/story/25520/break-displayedprice-into-two-single-price-components) for next steps 😊

Reply:
@FrancoAguzzi Thanks for that. We can consider this goal resolved for this PR then.